Library/ViewModels/CommentCellViewModel.swift

```diff
@@ -23,10 +23,16 @@ public protocol CommentCellViewModelOutputs {
   /// Emits text containing comment body.
   var body: Signal<String, Never> { get }
 
+  /// Emits a Bool determining if the bottomRowStackView is hidden.
+  var bottomRowStackViewIsHidden: Signal<Bool, Never> { get }
```

Comment:
Assuming the replyButton and flagButton should both be hidden we can just hide the entire stack view. Otherwise we'll see a larger-than-needed (read: unpleasant) padding at the bottom of the cell.